4.0.1/apache/gn-httpd.conf

```diff
@@ -0,0 +1,19 @@
+<VirtualHost *:80>
```

Comment:
There is also a nginx configuration below, why is there 2 different proxies to forward onto the webservices of the composition ?

Reply:
Because some projects are using Apache, some nginx. Only nginx is enabled in the composition.